packages/rocketchat-mongo-config/server/index.js

```diff
@@ -1,8 +1,19 @@
-import { Mongo } from 'meteor/mongo';
+import { Mongo, MongoInternals } from 'meteor/mongo';
 
 const mongoOptionStr = process.env.MONGO_OPTIONS;
 if (typeof mongoOptionStr !== 'undefined') {
 	const mongoOptions = JSON.parse(mongoOptionStr);
 
 	Mongo.setConnectionOptions(mongoOptions);
 }
+
+const isOplogEnabled = MongoInternals.defaultRemoteCollectionDriver().mongo._oplogHandle && !!MongoInternals.defaultRemoteCollectionDriver().mongo._oplogHandle.onOplogEntry;
```

Comment:
Curious.. should we do here or where all the other version checks and erroring out are located? 

https://github.com/RocketChat/Rocket.Chat/blob/develop/server/startup/serverRunning.js#L58

Reply:
Makes sense, totally forgot about this, thanks